catalog-info.yaml

```diff
@@ -137,7 +137,7 @@ spec:
         build_pull_requests: true # requires filter_enabled and filter_condition settings as below when used with buildkite-pr-bot
         publish_commit_status: false # do not update status of commits for this pipeline
         build_tags: false
-        build_branches: false
+        build_branches: true # enable build for branches. This relies on the filter in .buildkite/pipeline.elastic-agent-binary-dra.yml
```

Comment:
should we also add the `branch_configuration: "main 7.* 8.* v7.* v8.*"`

Reply:
Done in 6800ff27135e1013a8122eaad9479478562b7501